### ISSUE TITLE
[TW-1256] recurring tasks get duplicated on sync to taskd

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1290]
_Zed Jorarard on 2014-02-12T19:37:35Z says:_ 

 because i didnt test it myself for now the irc log:

<a> its seems that the task that appears on others machines as the next recur is uploaded too
<a> making 2 tasks with the same use
<a> also this seems to be related to the number of machines/users on a organization
<wopr> how?
<a> because the local taskwarrior adds the next task to recur and on sync it sends it back to taskserver as a new task

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2014-02-12T19:37:35Z
Modified: 2014-12-28T01:39:00Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2014-12-28T01:39:00Z says:_ 

 This is a taskd bug, represented by TD-44, and will not require any taskwarrior changes.  This will be fixed in Taskserver 1.1.0.
